docs/storage/multipathing.md

```diff
@@ -0,0 +1,425 @@
+---
+sidebar_position: 1
+---
+
+# Multipathing
+
+How to properly setup a new SR with multipathing on Xen Orchestra and XCP-ng.
+
+:::warning
+Do not attempt to enable multipathing on a production pool with existing and active iSCSI and/or HBA and/or FC SRs.
+:::
+
+## iSCSI
+
+### Requirements
+* Four different network interfaces (we recommend using two separate network cards)
+* Dedicated network interfaces without VLAN tagging on XCP-ng host and storage unit
```

Comment:
this is also not necessary, we have a ton of users and customers running iSCSI storage over vlans, there's nothing wrong with it at all